CODEOWNERS

```
@@ -29,6 +29,7 @@ bridges/otelslog                                                        @open-te
 bridges/otellogrus/                                                     @open-telemetry/go-approvers @dmathieu @pellared
 bridges/prometheus/                                                     @open-telemetry/go-approvers @dashpole
 bridges/otelzap/                                                        @open-telemetry/go-approvers @pellared @khushijain21
+bridges/otellogr/                                                       @open-telemetry/go-approvers @scorpionknifes
```

I can also be a codeowner

```suggestion
bridges/otellogr/                                                       @open-telemetry/go-approvers @scorpionknifes @pellared
```